lib/plenario_web/controllers/utils.ex

```diff
@@ -1,27 +0,0 @@
-defmodule PlenarioWeb.Controllers.Utils do
```

Comment:
This absolutely is necessary. Safari desktop doesn't have a date picker widget. We need to accommodate a handful of different ways of entering a date.